checkup.sh

```diff
@@ -0,0 +1,61 @@
+
+#!/bin/bash
+
+nodev=$(node -v);
+funcv=$(func -v);
+echo "Your node version is $nodev."
+echo "Your Azure Functions Core Tools version is $funcv."
+
+echo "--------------------------------";
+echo "Checking if node is installed...";
+echo "--------------------------------";
+if which node > /dev/null
+then
+    echo "Node is installed, skipping...";
+else
+    echo "Node is not installed.";
+    echo "Please install node here: https://nodejs.org/en/download/";
+    echo "Run this script again after to continue.";
+    exit
+fi
+
+echo "\n"
+echo "------------------------------------------------------";
+echo "Checking if Azure Functions Core Tools is installed...";
+echo "------------------------------------------------------";
+if which func > /dev/null
+then
+    if [[ ${funcv:0:1} == "2" || ${funcv:0:1} == "1" ]]
+    then
+        echo "Please upgrade your Azure Functions Core Tools version to version 4.";
+        echo "Follow this link to learn how to: https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=v4%2Clinux%2Ccsharp%2Cportal%2Cbash#changing-core-tools-versions";
+    fi
+    echo "Azure Function Core Tools is installed, skipping...";
+else
+    echo "Azure Function Core Tools is not installed.";
+    echo "Please install it here: https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=v4%2Clinux%2Ccsharp%2Cportal%2Cbash#install-the-azure-functions-core-tools";
+    echo "Run this script again after to continue.";
+    exit
+fi
+
+echo "\n"
+echo "---------------------------------------"
+echo "Checking if versions are compatible..."
+echo "---------------------------------------"
+if [[ (${nodev:1:2} == "16" || ${nodev:1:2} == "14") && (${funcv:0:1} == "4") ]]
+then
+    echo "Compatible versions! You are good to go.";
+elif [[ (${nodev:1:2} == "14" || ${nodev:1:2} == "12" || ${nodev:1:2} == "10") && (${funcv:0:1} == "3") ]]
```

Comment:
Node.js version 10 and 12 are out of maintenance. They should be excluded; maybe a message might make sense when checking the Node version that those are not supported